input/postgres/relation_stats.go

```diff
@@ -65,6 +66,44 @@ SELECT c.oid,
 			 AND c.oid NOT IN (SELECT pd.objid FROM pg_catalog.pg_depend pd WHERE pd.deptype = 'e' AND pd.classid = 'pg_catalog.pg_class'::regclass)
 			 AND n.nspname NOT IN ('pg_catalog','pg_toast','information_schema')
 			 AND ($1 = '' OR (n.nspname || '.' || c.relname) !~* $1)
+ UNION
+SELECT relid,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   NULL,
+	   NULL,
+	   NULL,
+	   NULL,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
```

Comment:
[Note for reviewer] I'm not a fan of this style, but don't see a better way to return this data, short of introducing something like `row_to_json` for the data (and deserializing on the client). Alternate ideas welcome :)

Reply:
agree with Maciek, I think it's okay to be like this

Reply:
Thanks, will leave as is then - I think its probably best to keep it simple for now.

Re:

> Separately, (and outside the scope of this patch) does locked_relids_with_parents not need to be a recursive CTE?

Good question - I don't think it needs to be a recursive CTE, because it exists to protect this part of the query, which is not recursive (i.e. only checks for direct children):

```sql
COALESCE((SELECT pg_catalog.sum(pg_catalog.pg_table_size(inhrelid))
  FROM pg_catalog.pg_inherits
  WHERE inhparent = c.oid), 0) AS size_bytes,
```